content/manifest-v2.md

```diff
@@ -305,8 +305,12 @@ addons:
 **variables** [Array, optional]: Describes variables.
 
 * **name** [String, required]: Unique name used to identify a variable. Example: `admin_password`
-* **type** [String, required]: Type of a variable. Currently supported variable types are `certificate`, `password`, `rsa`, and `ssh`. Example: `password`.
+* **type** [String, required]: Type of a variable. Currently supported variable types are `certificate`, `password`, `rsa`, and `ssh`.
 * **update_mode** [String, optional]: Update mode to use when generating credentials. Currently supported update modes are `no-overwrite`, `overwrite`, and `converge`. Defaults to `no-overwrite`. For further information about the behavior of the update modes, please refer to the [CredHub documentation](https://docs.cloudfoundry.org/api/credhub/version/2.9/#_overwriting_credential_values).
+* **update** [Hash, optional]:
+  * **strategy** [String, optional]: Controls when BOSH retrieves the latest version for the variable from the config server during a deploy. Currently supported strategies are `on-deploy`, and `on-stemcell-change`. Defaults to `on-deploy`.
+    * `on-deploy`: BOSH will look up the latest variable value on each deploy.
+    * `on-stemcell-change`: BOSH will only look up the latest variable value when a deploy is occuring that updates all stemcells for the deployment. If one or more stemcells are not being updated, then BOSH will continue to use the previously deployed value for the variable.
```

Comment:
Shouldn't we add the director version that supports this property?

Reply:
Great idea, we can update this when we have cut the release.

Elsewhere, I see the phrase `Available in bosh-release <VERSION>+.` so we'll probably add that in.